src/lib.rs

```diff
@@ -48,7 +49,6 @@ pub use ops::wrapping::{
 };
 pub use pow::{checked_pow, pow, Pow};
 pub use sign::{abs, abs_sub, signum, Signed, Unsigned};
-
```

Comment:
nit: this removal is unrelated, and I'd prefer to have that line for separation.